charts/private-action-runner/templates/deployment.yaml

```diff
@@ -28,9 +28,18 @@ spec:
           key: node
           operator: Equal
           value: nodeless
+      initContainers:
```

Comment:
issue was that [secrets volumeMount are always mounted readOnly ](https://github.com/kubernetes/kubernetes/issues/62099), so my workaround is initializing `/etc/dd-action-runner/` as empty and copying all files from secrets into it